buildchain/buildchain/constants.py

```diff
@@ -63,7 +64,15 @@
 
 K8S_VERSION  : str = '1.11.7'
 SALT_VERSION : str = '2018.3.4'
+KEEPALIVED_VERSION : str = '1.3.5-8.el7_6'
+KEEPALIVED_BUILD_ID : int = 1
 
+CENTOS_BASE_IMAGE : str = 'docker.io/centos'
+# pylint: disable=wrong-spelling-in-comment
+# Pylint doesn't like the spelling of `docker.io` ¯\_(ツ)_/¯
```

Comment:
You can add an entry in `.pylint-dict` instead of disabling it ;-)

Reply:
Done